macos/FluentUI/Core/Colors.swift

```diff
@@ -149,7 +149,7 @@ public final class Colors: NSObject {
 		case presenceUnknown
 
 		public var color: NSColor {
-			if let fluentColor = NSColor(named: "FluentColors/" + self.name, bundle: FluentUIResources.resourceBundle) {
+			if let fluentColor = NSColor(named: "FluentColors/" + self.name, bundle: FluentUIResources.colorsBundle) {
```

Comment:
if we have color bundle itself, then should the foldername "fluentColors" is even needed?

Reply:
It's still necessary because the existing builds for static lib + framework haven't changed. They still combine asset catalogs like before (which is why I need so many `#if` statements to keep the behavior separate).

If we want to change those too, then we could. But my goal for this PR was to have minimum risk for existing consumers and pipelines.